Scripts/Restriction/ranacompiler.py

```diff
@@ -103,9 +103,8 @@ def regex(site):
     """Construct a regular expression (string) from a DNA sequence.
 
     Example:
-
-        >>> regex('ABCGN')
-        'A[CGT]CG.'
+    >>> regex('ABCGN')
```

Comment:
Again, does this render OK. Does this work better?:

```rst
Examples
--------
>>> regex('ABCGN')
'A[CGT]CG.'
```

Reply:
OK